packages/accounts/src/light-account/account.test.ts

```diff
@@ -89,17 +88,13 @@ const givenConnectedProvider = ({
       owner,
       accountAddress: "0xb856DBD4fA1A79a46D426f537455e7d3E79ab7c4",
     },
-    client: createPublicErc4337FromClient(
-      createPublicClient({
-        transport: custom({
-          request: async ({ method }) => {
-            if (method === "eth_getStorageAt") {
-              return "0xae8c656ad28F2B59a196AB61815C16A0AE1c3cba"; // v1.1.0 address
-            }
-            return;
-          },
-        }),
-        chain,
-      })
-    ),
+    transport: custom({
+      request: async ({ method }) => {
+        if (method === "eth_getStorageAt") {
+          return "0xae8c656ad28F2B59a196AB61815C16A0AE1c3cba"; // v1.1.0 address
```

Comment:
what does this version refer to? v1.1.0

Reply:
ah yea it's light account version this address refers to